packages/addons-manager/bundle/config/upstream/crds/cni.tanzu.vmware.com_antreaconfigs.yaml

```diff
@@ -65,6 +65,22 @@ spec:
                 properties:
                   config:
                     properties:
+                      antreaProxy:
```

Comment:
@knabben - Looks like antrea configs were out of date. Do we need to port back to release-0.25 branch for TKGS?